src/status_im2/contexts/onboarding/new_to_status/view.cljs

```diff
@@ -0,0 +1,79 @@
+(ns status-im2.contexts.onboarding.new-to-status.view
+  (:require
+   [quo2.components.markdown.text :as text]
+   [quo2.components.onboarding.small-option-card.view :as small-option-card]
+   [quo2.core :as quo]
+   [re-frame.core :as re-frame]
```

Comment:
We use re-frame as `rf`. And import from `[utils.re-frame :as rf]`

Reply:
I've seen that we're not always importing `utils.re-frame` to call a `dispatch` :thinking:  do we have a convention here?

Solved in e9768b8495f1c4cda2457a6442e6834ad493c2fa